dp-frontend-renderer.nomad

```diff
@@ -0,0 +1,82 @@
+job "dp-frontend-renderer" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger      = "20s"
+    max_parallel = 1
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    task "dp-frontend-renderer-web" {
+      driver = "docker"
+
+      config {
+        image = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-frontend-renderer/{{REVISION}}.tar.gz"
+
+        port_map {
+          http = 8080
+        }
+      }
+
+      service {
+        name = "dp-frontend-renderer"
+        port = "http"
+        tags = ["web"]
+      }
+
+      resources {
+        cpu    = "{{WEB_RESOURCE_CPU}}"
+        memory = "{{WEB_RESOURCE_MEM}}"
+
+        network {
+          port "http" {}
+        }
+      }
+    }
+  }
+
+  group "publising" {
+    count = "{{PUBLISHING_TASK_COUNT}}"
```

Comment:
Doesn't look like these values have been substituted by the build. Have we defined the resources for the publishing tasks?